thealgorithms/project_euler/problem_36/sol1.py

```diff
@@ -8,23 +8,27 @@
 
 (Please note that the palindromic number, in either base, may not include leading zeros.)
 '''
-try:
-	xrange		#Python 2
-except NameError:
-	xrange = range	#Python 3
+def main():
+    try:
+	    xrange		#Python 2
```

Comment:
Double tab here?

Reply:
No but i was using spaces when the original code had tabs. I changed my spaces into tabs, should look consistent now